File: Lenguajes_Programacion/PHP/6_clases.php
<?php
// Clases
class MyClass {
    public $name;
    public $age;

    function __constructor($name, $age) {
        $this->name = $name;
        $this->age = $age;
    }
}

$my_class = new MyClass("Diego", 23); // Definimos una variable para crear una instancia de una clase.
print_r($my_class); // Imprime la estructura de la clase
echo $my_class->name . "\n"; // "Diego"

$my_class->name = "Brais"; // Se cambia el valor de la variables
echo $my_class->name . "\n"; // Imprime el valor cambiado
echo gettype($my_class) . "\n"; // Type Object

File: Lenguajes_Programacion/9_Rust/1_Sintaxis_Rust/6_clases.rs
fn main() {

    // Instaciando una estructura
    let my_struct = MyStruct:: new("Diego", 23);
    println!("{} {}", my_struct.name, my_struct.age);

}

// Estructuras. Funciona como clases

struct MyStruct {
    name : String,
    age: i32,
}

impl MyStruct {
    fn new(name : &str, age: i32,) -> MyStruct{
        MyStruct {
            name :String::from(name),
            age
            }
    }
}